Test2/Tests/SettingPageBase.cs

```diff
@@ -0,0 +1,150 @@
+        }
+
+        [TestCase("dd/MM/yyyy")]
+        [TestCase("dd-MM-yyyy")]
+        [TestCase("MM/dd/yyyy")]
+        [TestCase("MM dd yyyy")]
+
+        public void DateFormat(string format)
+        {
+            IWebElement dateForm = driver.FindElement(By.XPath("//select[@id='dateFormat']"));
+            SelectElement dateFormSeletct = new SelectElement(dateForm);
+            dateFormSeletct.SelectByText(format);
+            IWebElement btnSave = driver.FindElement(By.Id("save"));
+            btnSave.Click();
+            WaitForAlert();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.TitleContains("Deposite"));
+            IWebElement endDate = driver.FindElement(By.Id("endDate"));
+            string expectedDate = DateTime.Now.ToString(format);
+            Assert.AreEqual(expectedDate, endDate.GetAttribute("value"));
+        }
+
+        [TestCase("$ - US dollar", "$")]
+        [TestCase("€ - euro", "€")]
+        [TestCase("£ - Great Britain Pound", "£")]
+        [TestCase("₴ - Ukrainian hryvnia", "₴")]
+
+        public void Currency(string curren,string simcur)
+        {
+            IWebElement currenc = driver.FindElement(By.XPath(".//select[@id='currency']"));
+            IWebElement btnSave = driver.FindElement(By.Id("save"));
+            SelectElement currencSelect = new SelectElement(currenc);
+            currencSelect.SelectByText(curren);
+            WaitForReady();
+            btnSave.Click();
+            WaitForAlert();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.UrlContains("Calculator"));
+            string ActualUrl = driver.Url;
+            string expectedUrl = "https://localhost:5001/Calculator";
+            Assert.AreEqual(expectedUrl, ActualUrl);
+            IWebElement simb = driver.FindElement(By.Id("currency"));
+            Assert.AreEqual(simcur, simb.Text);
+        }
+
+        [Test]
+
+        public void Logut()
+        {
+            IWebElement logut = driver.FindElement(By.XPath("//div[@class='login link btn btn-link']"));
+            logut.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.TitleContains("Login"));
+            string ActualUrl = driver.Url;
+            string expectedUrl = "https://localhost:5001/";
+            Assert.AreEqual(expectedUrl, ActualUrl);
+        }
+
+        [TestCase("123,456,789.00", "200,000.00", "100,000.00")]
+        [TestCase("123.456.789,00", "200.000,00", "100.000,00")]
+        [TestCase("123 456 789.00", "200 000.00", "100 000.00")]
+        [TestCase("123 456 789,00", "200 000,00", "100 000,00")]
+
+        public void NumbForm(string formn, string forminc, string forminter)
```

Comment:
Variables should have better names, so it is easier to understand their role in the test.
```suggestion
        public void NumbForm(string format, string expectedIncome, string expectedInterest)
```